package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dohnut",
-  "version": "4.6.0",
+  "version": "4.6.1",
```

Comment:
You can leave the version bump out of the PR. I'll do that separately when any other changes (if any) at time of `npm publish`.

Reply:
I think this would still be a minor version bump since some behaviour changes in a (hopefully) backwards compatible manner.

Reply:
I don't think it should break anything existing, but I'll leave that to you. I took out the version bump in the next commit.